README.md

```diff
@@ -226,7 +226,7 @@ http://docs.ansible.com/
 ## Roadmap
 Cross platform compatibility (Ubuntu, CentOS)
 
-Greater diversity of interchangeable roles (nginx/apache, unicorn/puma/passenger, rvm/rbenv)
+Greater diversity of interchangeable roles (nginx/apache, unicorn/puma/passenger)
```

Comment:
Is this actually something users are asking for?

Reply:
Not really 😌